data_steward/cdr_cleaner/cleaning_rules/map_health_insurance_responses.py

```diff
@@ -16,68 +27,57 @@
 ORIGINAL_OBSERVATION_SOURCE_CONCEPT_ID = 43528428
 HCAU_OBSERVATION_SOURCE_CONCEPT_ID = 1384450
 
-ISSUE_NUMBERS = ['dc826']
+ISSUE_NUMBERS = ['DC826', 'DC2746']
 
 INSURANCE_LOOKUP = 'insurance_lookup'
 NEW_INSURANCE_ROWS = 'new_insurance_rows'
-PIDS_LOOKUP_TABLE = 'health_insurance_pids'
-PIPELINE_TABLES = 'pipeline_tables'
-
-INSURANCE_LOOKUP_FIELDS = [{
-    "type": "string",
-    "name": "answer_for_obs_src_c_id_43528428",
-    "mode": "nullable",
-    "description": "Answers for observation_source_concept_id = 43528428"
-}, {
-    "type": "string",
-    "name": "basics_value_source_value",
-    "mode": "nullable",
-    "description": "value_source_value field for the basics survey answer"
-}, {
-    "type": "integer",
-    "name": "basics_value_source_concept_id",
-    "mode": "nullable",
-    "description": "value_source_concept_id for the basics survey answer"
-}, {
-    "type": "string",
-    "name": "hcau_value_source_value",
-    "mode": "nullable",
-    "description": "value_source_value field for the HCAU survey answer"
-}, {
-    "type": "integer",
-    "name": "hcau_value_source_concept_id",
-    "mode": "nullable",
-    "description": "value_source_concept_id for the HCAU survey answer"
-}]
-
-SANDBOX_CREATE_QUERY = JINJA_ENV.from_string("""
-CREATE TABLE `{{project_id}}.{{sandbox_dataset_id}}.{{new_insurance_rows}}`
+SANDBOXED_INSURANCE_ROWS = 'sandboxed_insurance_rows'
+HEALTH_INSURANCE_PIDS = 'health_insurance_pids'
```

Comment:
**Acceptance Criteria**
- Table names that will not change should be moved to curation/data_steward/common.py and imported as needed.

These are table name constants that don't change and should be moved to common.py

Reply:
I agree with 'health_insurance_pids' as it is in pipeline_tables, but am not sure about the others. 
It does not look like we usually put intermediary tables, sandbox tables, or curation/resource_files/ tables in common.py?